testdata/go/v4-alpha/monitoring/memcached-operator/monitoring/metricsdocs/metricsdocs.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/example/memcached-operator/monitoring"
+)
+
+// please run \"make generate-metricsdocs\" to generate this document.
+// constant parts of the file
+const (
+	title      = "# Memcached Operator Metrics\n"
+	background = "This document aims to help users that are not familiar with metrics exposed by the memcached-operator.\n" +
+		"All metrics documented here are auto-generated by the utility tool \"monitoring/metricsdocs\" and reflects exactly what is being exposed.\n\n"
```

Comment:
>"All metrics documented here are auto-generated by the utility tool \"monitoring/metricsdocs\" and reflects exactly what is being exposed.\n\n"

"The metrics documentation is auto-generated by the utility tool \"monitoring/metricsdocs\" and reflects all of the metrics that are exposed by the operator.\n\n"

Reply:
Agree

Reply:
updated